### DISCUSSION TITLE
DCO requirement

### DISCUSSION THREAD

**Discussion by nlohmann** [nlohmann/json#4578] [Announcements]
As of January 1, 2025, the project will require DCO (https://developercertificate.org) for all merge requests.

**Comment by u-235** [comment]
Seems reasonable, although somewhat redundant. Should the text of the certificate be attached to the PR or should the certificate be placed in the commit comment?

**Reply by nlohmann** [reply]
A sign-off is sufficient. There is a GitHub Action in place that checks this.

**Comment by smoken64** [comment]
Will the DCO also be added to my azure pipelines?

**Reply by nlohmann** [reply]
What do you mean? This project is not using Azure.

**Reply by smoken64** [reply]
Ok I understand now 👍
